src/app/component/dashboard/customers/customers.component.scss

```diff
@@ -86,4 +85,14 @@ html, body {
 .copyPatientForm {
   position: absolute;
   bottom: 10px;
-}
\ No newline at end of file
+}
+
+.hide-b{
```

Comment:
Ove css klase se pojavljuju i vise komponenti i ovo nije dobro.

- pravimo sebi bespotreban posao prilikom razvoja
- odrzavanje je teze, jer u slucaju da zelimo da menjamo nesto, moramo to da radimo na vise razlicitih mesta

Ukoliko neki delovi koda uticu na vise razlicitih komponenti, oni moraju da budu genericki i da se nalaze u globalnim komponentama. U ovom slucaju u style.css(scss) klasi, ukoliko naravno ove klase ne uticu na komponente na kojima ne bi trebale da uticu.